### PR TITLE
imp: install PHP-CS-Fixer locally, allowing us to run in locally

### PR DESCRIPTION
**Pull request by Kocal**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no 
| Tickets       | Fix #... 
| License       | MIT

After opening #616, the CI showed me a lot of style issues in the PHP code: 
image

I don't have PHP CS Fixer globally installed on my machine because that's a bad practice, and I don't want to fix those errors manually (the time is precious).

So, this PR adds PHP CS Fixer locally.

### CODE REVIEW

**Review by kbond — APPROVED** [review]
Makes sense.

File: composer.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
     "name": "symfony/ux",
     "license": "MIT",
     "require-dev": {
-        "symfony/filesystem": "^5.2|^6.0"
+        "symfony/filesystem": "^5.2|^6.0",
+        "friendsofphp/php-cs-fixer": "^3.13"
```
Comment (kbond) [inline composer.json]: Will probably never be an issue here but might be better to use `php-cs-fixer/shim` to avoid any possible dep conflicts.

Comment (Kocal) [reply composer.json]: Oh yeah, I forgot about this package.
Gonna change it ASAP.

Comment (Kocal) [reply composer.json]: Done

**Comment by weaverryan** [comment]
I think the recommended way is to install it into a `tools/` directory, right? https://github.com/PHP-CS-Fixer/PHP-CS-Fixer#installation

**Comment by kbond** [comment]
The shim is new maybe? I only learned of it from the recipe (https://github.com/symfony/recipes/pull/1112) that was merged yesterday.

**Comment by weaverryan** [comment]
This was the first I had heard of it. It seems like a good idea - is there some docs or mention of using this? And how it fits into best-practices?

**Comment by Kocal** [comment]
As you know, installing this kind of tools into a separate composer.json (ie: in `tools/...` directory) allows to not pollute your app's deps/dev-deps with the tool's dependencies.

However I'm not a big fan of having multiple `composer.json` in a single project, and you can face some problems if you try to load/interpret your code with the tool (I remember to face this kind of issue because the are two autoloaders, but I may be wrong). 

This is something that can be _easily_ fixed by using a .phar, which can be installed through a shim-version of the package. 

To me the given installation methods for PHP-CS-Fixer are "wrong", `composer require --dev friendsofphp/php-cs-fixer` should download a `.phar` and that's all, no other dependencies, like PHPStan and Rector do since a long time.

**Comment by Kocal** [comment]
Hi @weaverryan @kbond, is installing the shim still okay?

**Review by kbond — APPROVED** [review]
Works for me.

**Comment by weaverryan** [comment]
This *does* make a lot of sense - thanks Hugo!

**Pull request merged**